### ISSUE TITLE
Redux Connect

### ISSUE THREAD

**Issue by ghost** [xabikos/vscode-react#32]
Hi, I downloaded the extension v1.4.0 and the Redux Connect (rrc) snippet isn't there?

**Comment by JamesTheHacker** [comment]
Same problem no `rrc` snippet. The rest work ok.

**Comment by JamesTheHacker** [comment]
Same problem. For now I solved it by adding a JavaScript user snippet:

```
  "reactReduxComponent": {
    "prefix": "rrc",
    "body": "import React, { Component } from 'react';\nimport { connect } from 'react-redux';\n\nfunction mapStateToProps(state) {\n\treturn {\n\n\t};\n}\n\nclass ${1:componentName} extends Component {\n\trender() {\n\t\treturn (\n\t\t\t<div>\n\t\t\t\t$0\n\t\t\t</div>\n\t\t);\n\t}\n}\n\nexport default connect(\n\tmapStateToProps,\n)(${1:componentName});",
    "description": "Creates a React component class connected to redux"
  }
```

Works. Not sure why it's a problem in this package though. Will look into it and submit PR if I can find the problem.

**Comment by G2Jose** [comment]
Looks like the [published version of the extension](https://marketplace.visualstudio.com/items?itemName=xabikos.ReactSnippets) does not contain the same code as v1.4.0 of this repo. Searching for `rrc` in `~/.vscode/extensions/xabikos.reactsnippets-1.4.0/snippets/snippets.json` gives no results. 

Maybe a publishing issue @xabikos ?

Until a newer version is published, anyone who wants the latest version can replace the contents of `~/.vscode/extensions/xabikos.reactsnippets-1.4.0/snippets/snippets.json` with code from https://raw.githubusercontent.com/xabikos/vscode-react/master/snippets/snippets.json

**Comment by xabikos** [comment]
@johnyousif @JamesTheHacker @G2Jose You are absolutely right and my apologies for the late response. 
I merged the PR #30 but didn't publish a new version, something that I [just did](https://marketplace.visualstudio.com/items?itemName=xabikos.ReactSnippets).

I am closing this but please feel free to open again if there is a need.
